crates/relayer-framework/src/lib.rs

```diff
@@ -5,6 +5,12 @@
 
 extern crate alloc;
 
+#[cfg(test)]
+extern crate std;
+
+#[cfg(test)]
+pub mod tests;
```

Comment:
Looking at it now, I think it might be better if we create a dedicated `relayer-framework-test` crate for this. It would be better if we can avoid using feature flags and dev dependencies to keep the `relayer-framework` crate itself clean of all possible depedencies.